lib/remotus/winrm_connection.rb

```diff
@@ -69,14 +70,16 @@ def base_connection(reload: false)
 
     #
     # Retrieves/creates the WinRM shell connection for the host
+    #
+    # @param [symbol] type command type
     # If the connection already exists, the existing connection will be retrieved
     #
-    # @return [WinRM::Shells::Powershell] remote connection
+    # @return [WinRM::Shells::type] remote connection
     #
-    def connection
+    def connection(type = :powershell)
```

Comment:
Rename the type parameter to shell.

```suggestion
    def connection(shell = :powershell)
```